index.html

```diff
@@ -292,7 +293,7 @@ <h3 id="ruby-character-size-and-character-placement">Ruby character size and cha
 <li id="l20200529016">In vertical text, ruby annotation is placed to the right of the base characters,
     and the character frame of the ruby annotation is placed flush
     against the character frame of the base characters.
-	
+	(See also JLReq <a href="https://www.w3.org/TR/jlreq/#kanji_hiragana_and_katakana">section 2.1.2 Kanji, Hiragana and Katakana</a> for details on character frame.)
```

Comment:
and here:

(the '<a href="https://www.w3.org/TR/jlreq/#kanji_hiragana_and_katakana">character frame</a>' is described in JLReq)